docs/datamodel/extensions.rst

```diff
@@ -8,10 +8,66 @@ Extensions are the way EdgeDB adds more functionality. In principle,
 extensions could add new types, scalars, functions, etc., but, more
 importantly, they can add new ways of interacting with the database.
 
+Built-in extensions
+-------------------
+
+There is a few built-in extensions available:
+
 - ``edgeql_http``: enables :ref:`EdgeQL over HTTP <ref_edgeql_http>`
 - ``graphql``: enables :ref:`GraphQL <ref_graphql_index>`
 - ``auth``: enables :ref:`EdgeDB Auth <ref_guide_auth>`
```

Comment:
I know you haven't changed these, but we should add the other extensions here while we're at it. pgcrypto, et al.